content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,241 @@
+
+<br />
+<a href="https://docs.sourcegraph.com/integration"><Badge text="Extensions" color="vermillion" size="small" /></a>
+
+#### Your favorite extensions are now available by default 
+Top used extensions, including code navigation, git-extras, open-in-editor, and search-exports, are now part of the platform and no longer require you to  enable on your own. Essential code context is surfaced during the ideal moments in your workflow and knowledge about the codebase from the extensions can be found in the sidebar, when hovering over code, and after clicking on "more actions."
+
+Now that our top extensions are part of the core functionality, there will be greater support and functionality over time. With this change, you will no longer be able to access the extensions registry or create extensions on a private registry. Note: if you still need access to extensions or create extensions on a private registry, you can enable a feature flag to do so until early 2023.
+
+This update *does not* impact our IDE extensions, which will continue to allow you to search and navigate across all of your repositories without ever leaving your IDE or checking them out locally.
+
+<div className="tw-mb-sm">
+  <Figure
+    src="https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/Blame.png"
+    alt="Git Blame"
+  />
+</div>
+
+<br />
+<a href="/code-search"><Badge text="Code Search" color="cerise" size="small" /></a>
+
+#### Quickly access answers within your codebase with a revamped reference panel 
+Your code navigation experience just got better with a refreshed version of the reference panel. Stay within your workflow with a new preview panel that helps you navigate references without having to leave the file. The consolidation of definitions, references, and implementations in the same tab reduces the number of clicks needed to get pertinent information. You can find the references you're looking for much faster with new filters instead of having to scroll through the page, which is particularly challenging for navigating symbols with a large number of results.
+
+<Video 
+  source={{
+    webm: 'blog/release-post/4.0/Reference%20panel',
+    mp4: 'blog/release-post/4.0/Reference%20panel'
+  }}
+  loop={true}
+  title="Reference panel refresh"
+  caption="Stay within your workflow and quickly access references with the revamped reference panel"
+/>
+
+<a href="https://docs.sourcegraph.com/code_navigation/explanations/features#find-references" className="tw-not-italic tw-flex tw-items-center">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+
+
+### 🏗️ High-leverage ways to improve your entire codebase
+<br />
+<a href="/batch-changes"><Badge text="Batch Changes" color="blue" size="small" /></a>
+
+#### Make changes across your codebase at enterprise scale with server-side Batch Changes (beta)
```

Comment:
@malomarrec you please review this by EOD today to ensure the content is accurate? Please keep in mind that we're focusing on the benefits and value to our customers/prospects within this release post.